examples/topography_spectra.jl

```diff
@@ -0,0 +1,129 @@
+using LazyArtifacts
+using ClimaAtmos.AtmosArtifacts
+import ClimaAtmos as CA
+import ClimaCore as CC
+import ClimaComms
+import ClimaParams as CP
+import ClimaCore.Quadratures
+using ClimaUtilities: SpaceVaryingInputs
+using ClimaCore.Hypsography: diffuse_surface_elevation!
+using ClimaCore: Remapping, Geometry, Operators, Spaces
+using CairoMakie
+
+import ClimaCoreSpectra: power_spectrum_2d
+
+const AA = AtmosArtifacts
+
+
+function mask(x::FT) where {FT}
+    return x * FT(x > 0)
+end
+
+"""
+    generate_spaces(; h_elem=60, diffiter=32)
+For a given number of elements `h_elem`, this function generates a 
+horizontal spectral space (ClimaCore.SpectralElementSpace2D) and
+generates an interpolated (using SpaceVaryingInputs) orography field
+on this space. The `diffuse_surface_elevation` function is called 
+to apply `diffiter` instances of Laplacian diffusion passes 
+(which rely on the canonical unsteady diffusion equation).
+Returns a `Field` containing surface orography. 
+"""
+function generate_spaces(; h_elem = 60, diffiter = 32, planet_radius = 6.378e6)
+    FT = Float32
+    cubed_sphere_mesh =
+        CA.cubed_sphere_mesh(; radius = FT(planet_radius), h_elem)
+    quad = Quadratures.GLL{4}()
+    comms_ctx = ClimaComms.context()
+    h_space = CA.make_horizontal_space(cubed_sphere_mesh, quad, comms_ctx, true)
+    Δh_scale = Spaces.node_horizontal_length_scale(h_space)
+    @assert h_space isa CC.Spaces.SpectralElementSpace2D
+    coords = CC.Fields.coordinate_field(h_space)
+    target_field = CC.Fields.zeros(h_space)
+    elev_from_file = SpaceVaryingInputs.SpaceVaryingInput(
+        AA.earth_orography_file_path(; context = comms_ctx),
+        "z",
+        h_space,
+    )
+    elev_from_file = @. mask(elev_from_file)
+    diffuse_surface_elevation!(
+        elev_from_file,
+        κ = FT(28e7 * (30 / h_elem)^2),
```

Comment:
```suggestion
        κ = diff_courant * Δh_scale^2 / dt,
```
Please avoid arbitrary numbers like 28e7 in code. I think what you want here is something like the above, where diff_courant is some parameter to ensure numerical stability (e.g., 0.1).

Reply:
Noted, this value is from the E3SM docs which I can make clearer - in general I'll replace this (via changes in `ClimaCore` to depend entirely on resolution and required attenuation factors) - (following the Slack discussion)